### PR TITLE
Add dependencies for weasyprint and pdf2image

### PR DESCRIPTION
**Pull request by aru31**

### CODE REVIEW

**Comment by dhruvkb** [comment]
Finally it comes to this @shaddygarg.

File: django/Dockerfile [hunk 1]
```diff
@@ -11,9 +11,13 @@ ARG IMAGETYPE
 RUN groupadd --system django && useradd --system --gid django django
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get install -y gcc \
-    && rm -rf /var/lib/apt/lists/*
+ RUN apt-get update \
+     && apt-get install -y gcc build-essential \
+     && apt-get install -y python3-cffi libcairo2 \
+     python3-cffi libcairo2 libpango-1.0-0 \
```
Comment (dhruvkb) [inline django/Dockerfile]: Put all of these in one line.

File: django/Dockerfile [hunk 2]
```diff
@@ -11,8 +11,9 @@ ARG IMAGETYPE
 RUN groupadd --system django && useradd --system --gid django django
 
 # Install dependencies
-RUN apt-get update \
-    && apt-get install -y gcc \
+ RUN apt-get update && apt-get install -y gcc build-essential && apt-get install -y python3-cffi libcairo2 python3-cffi libcairo2 libpango-1.0-0 \
+    libpangocairo-1.0-0 libgdk-pixbuf2.0-0 \
+    libffi-dev shared-mime-info \
```
Comment (dhruvkb) [inline django/Dockerfile]: Why the weird line breaks? Put each command on its own line and then break lines only when the line crosses 80 characters.

Comment (dhruvkb) [reply django/Dockerfile]:
```Dockerfile
RUN apt-get update
    && apt-get install -y <all your packages> \
        <more packages> \
        <even more packages>
```

For best results sort packages alphabetically.

**Comment by pradumangoyal** [comment]
@aru31 @shreyansh23 can you please check the review. The time is near when we need to pull it on our production server. 😁

**Comment by pradumangoyal** [comment]
@dhruvkb Can you check the changes, please? Are they in the order suggested by you?

**Comment by dhruvkb** [comment]
Will do by EOD.

**Comment by pradumangoyal** [comment]
@dhruvkb Reminder 😅

**Comment by pradumangoyal** [comment]
Make sure one squash this PR before merging.

**Pull request merged**